sky/utils/command_runner.py

```diff
@@ -223,8 +224,9 @@ def run(
         command += [
             shlex.quote(f'true && source ~/.bashrc && export OMP_NUM_THREADS=1 '
                         f'PYTHONWARNINGS=ignore && ({cmd})'),
-            '2>&1',
         ]
+        if not separate_stderr:
```

Comment:
If `separate_stderr` is True, is there a way to get the stderr? If not, should we rename this arg to `exclude_stderr` so its clear you can't retrieve the stderr?